pkg/scheduler/scheduler.go

```diff
@@ -326,7 +326,8 @@ func (sched *Scheduler) preempt(preemptor *v1.Pod, scheduleErr error) (string, e
 				klog.Errorf("Error preempting pod %v/%v: %v", victim.Namespace, victim.Name, err)
 				return "", err
 			}
-			sched.config.Recorder.Eventf(victim, v1.EventTypeNormal, "Preempted", "by %v/%v on node %v", preemptor.Namespace, preemptor.Name, nodeName)
+			sched.config.Recorder.Eventf(victim, nil, v1.EventTypeNormal, "Preempted", "", "Preempted by %v/%v on node %v", preemptor.Namespace, preemptor.Name, nodeName)
```

Comment:
I actually believe, that we should set "related" to preemptor
[it's exactly the example of case for which related was created].